### PR TITLE
Update to 0.8

### PR DESCRIPTION
**Pull request by ethanfrey**
Update to newly tagged v0.8.0 cosmwasm.

### CODE REVIEW

**Comment by ethanfrey** [comment]
Tested manually and everything checks out, but it would be nice to have CI.

Plus, I will have to update this a few times with newer cosmwasm versions

File: Cargo.toml [hunk 1]
```diff
@@ -30,17 +30,18 @@ overflow-checks = true
...
+cosmwasm-storage = { git = "https://github.com/CosmWasm/cosmwasm", rev = "7363cd83d7168ff867a7707dcc9a4acd1d7bbce0" }
+schemars = "0.7"
+serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+snafu = { version = "0.6.3" }
 
 [dev-dependencies]
-cosmwasm-vm = { version = "0.7.2", default-features = false }
+cosmwasm-vm = { git = "https://github.com/CosmWasm/cosmwasm", default-features = false, rev = "7363cd83d7168ff867a7707dcc9a4acd1d7bbce0" }
+cosmwasm-schema = { git = "https://github.com/CosmWasm/cosmwasm", rev = "7363cd83d7168ff867a7707dcc9a4acd1d7bbce0" }
 serde_json = "1.0"
```
Comment (webmaster128) [inline Cargo.toml:47]: See https://github.com/CosmWasm/cosmwasm/blob/master/MIGRATING.md#update-schema-code ;)

Comment (ethanfrey) [reply Cargo.toml:47]: ahh. serde_json. gotcha

File: examples/schema.rs [hunk 2]
```diff
@@ -18,29 +17,3 @@ fn main() {
     export_schema(&schema_for!(State), &out_dir);
     export_schema(&schema_for!(CountResponse), &out_dir);
 }
```
Comment (webmaster128) [inline examples/schema.rs:19]: You can add `remove_schemas` now (https://github.com/CosmWasm/cosmwasm/pull/282/files#diff-d9cf9a2c5387a424a227294fa17aac38)

**Comment by ethanfrey** [comment]
Let's wait to merge until we have 0.8 final for cosmwasm

**Comment by webmaster128** [comment]
Jap. I wonder where out CI is gone … Let's see

**Comment by webmaster128** [comment]
Arg, Windows build error since blake3 v0.1.5 -> v0.3.4 (pulled in by the Wasmer 0.17 update)

**Comment by ethanfrey** [comment]
We don't need to support Windows (only OSX and Linux can run the chain).
How about removing windows from the CI and then making an issue to fix this soon?

**Comment by webmaster128** [comment]
> We don't need to support Windows (only OSX and Linux can run the chain).
> How about removing windows from the CI and then making an issue to fix this soon?

We use AppVeyor in order to be able to differentiate between meta CI config and the resulting project's CI config. AppVeyor uses Windows by default. We could try to reactivate https://github.com/CosmWasm/cosmwasm-template/pull/17 in order to use AppVeyor+Ubuntu.

**Comment by ethanfrey** [comment]
Oof, this is not a quick fix.

I'll run the tests locally and then merge?

**Comment by ethanfrey** [comment]
And revisit the CI after merge

**Comment by ethanfrey** [comment]
I tested this locally and it works. Feel free to merge @webmaster128

**Pull request merged**